### ISSUE TITLE
TypeError: t is not a function

### ISSUE THREAD

**Issue by rodneyrehm** [hilongjw/vue-lazyload#227]
After upgrading to [v1.2.0](https://github.com/hilongjw/vue-lazyload/releases/tag/v1.2.0) I get the following errors logged to the console:

```
Uncaught TypeError: t is not a function
    at t.value (vue-lazyload.js:6)
    at vue-lazyload.js:6
    at Array.forEach (<anonymous>)
    at vue-lazyload.js:6
    at Array.forEach (<anonymous>)
    at c.value (vue-lazyload.js:6)
value @ vue-lazyload.js:6
(anonymous) @ vue-lazyload.js:6
(anonymous) @ vue-lazyload.js:6
value @ vue-lazyload.js:6

Uncaught TypeError: t is not a function
    at vue-lazyload.js:6
    at Image.r.onload (vue-lazyload.js:6)
```

Unfortunately there's no SourceMap provided with the built file, but loading `vue-lazyload/src/index` provides:

```
Uncaught TypeError: onFinish is not a function
    at ReactiveListener.load (listener.js?4872:140)
    at ListenerQueue.forEach.listener (lazy.js?c2ca:364)
    at Array.forEach (<anonymous>)
    at entries.forEach.entry (lazy.js?c2ca:361)
    at Array.forEach (<anonymous>)
    at Lazy._observerHandler (lazy.js?c2ca:359)

Uncaught TypeError: onFinish is not a function
    at data (listener.js?4872:159)
    at Image.image.onload (util.js?4b80:203)
```

Additionally a resource that couldn't be loaded is now logged to the console:

```
Error { type: "error", target: img, … }
    at err (listener.js?4872:161)
    at image.onerror (util.js?4b80:212)
```

I tried setting up a minimal test-case, but I can't get vue-lazyload to work via unpkg… https://jsfiddle.net/d50bkwxj/1/

--- 

I also noticed there are a bunch of files in the npm module that aren't required (`.circleci`, `.babelrc`, `test`, …) - see https://docs.npmjs.com/files/package.json#files

**Comment by hilongjw** [comment]
@rodneyrehm , I checked the [demo](https://jsfiddle.net/d50bkwxj/1/) you provided, and found that the referenced `vue-lazyload` was not correct.

```html
<script src="https://unpkg.com/vue"></script>
<!-- ✅ -->
<script src="https://unpkg.com/vue-lazyload"></script>

<!-- ❌ -->
<script src="https://unpkg.com/vue-lazy-load"></script>
<div id="app">
  <img v-lazy="image" alt="demo">
</div>

```

```javascript
// ✅VueLazyload
// ❌vueLazyLoad

Vue.use(VueLazyload, {
  // set observer to true
  observer: true,
  observerOptions: {
    rootMargin: '0px',
    threshold: 0.1,
  }
})
```

**Comment by hilongjw** [comment]
I've fixed this problem at 1.2.1, and it seems that there is no problem in  [my demo](https://jsfiddle.net/Awee/wqLvjLwo/)

**Comment by rodneyrehm** [comment]
> the referenced `vue-lazyload` was not correct.

I'm sorry for mixing that up. I probably should've gone to bed before opening that issue :(

> 've fixed this problem at 1.2.1

I can confirm the problem is fixed. thank you very much for the quick response! :)
